go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/twitchtv/twirp
```

Comment:
this will not work because Twirp is tagged beyond v1. See https://github.com/twitchtv/twirp/issues/169 for an extended discussion

Reply:
Have not tested this, but wouldn't this work with the `+incompatible` tag in the `go.mod`?

```
module example.com/twirp-svc

require github.com/twitchtv/twirp v5.6.0+incompatible
```

Reply:
It will not, because `+incompatible` indicates that the module does not have a go.mod file...once the repository has a go.mod file then you cannot opt-out of Semantic Import Versioning